src/fn/edit-contacts.js

```diff
@@ -34,16 +52,29 @@ const execute = () => {
         const nameParts = fs.path.basename(csv).split('.');
         const prefix = nameParts[0];
         switch(prefix) {
-          case 'contact':  return processDocuments(csv, getIDs(csv, prefix), db, args);
-          case 'users': return processUsers(csv, getIDs(csv, prefix), db, args);
+          case 'contact':  return processDocuments('contact', csv, getIDs(csv, prefix), db, args);
+          case 'users': return processDocuments('user', csv, getIDs(csv, prefix), db, args);
           default: throw new Error(`Unrecognised CSV type ${prefix} for file ${csv}`);
         }
       })
       .then(docs => addToModel(csv, docs)),
       Promise.resolve())
 
     .then(() => model.exclusions.forEach(toDocs.removeExcludedField))
-    .then(() => Promise.all(Object.values(model.docs).map(saveJsonDoc)));
+    .then(() => Promise.all(Object.values(model.docs).map(doc => {
+      const saveDoc = saveJsonDoc(doc, args, overwriteAllFiles);
+
+      if(saveDoc === DOC_EXISTS) {
+        const userSelection = userPrompt.keyInSelect();
+
+        if (userSelection === undefined || userSelection === 2) {
```

Comment:
Watch out for the [magic number code smell](https://maximilianocontieri.com/code-smell-02-constants-and-magic-numbers) here.

For here and the other similar usages, please replace with a named constant so future readers can think about it logically, without needing to figure out what a "1" might mean.